nixos/tests/amazon-image.nix

```diff
@@ -0,0 +1,60 @@
+# Tests building and running a GUID Partition Table (GPT) appliance image.
```

Comment:
Nothing about this test, except maybe the sectorSize is amazon-specifc, anymore. Its a generic test for repart-built gpt images.